### PR TITLE
ENH: Convert Images to HU by Default

### LINKED ISSUE #41: Convert to Hounsfield units by default
Currently, the reader does not do conversion to HU. We should enabled that by default.

### PR DESCRIPTION
**Pull request by mkuczyns**
The images read by the ScancoImageIO reader are not converted to HU by default. This PR should fix this issue and has been tested with AIM images of type short and ISQ images. Closes #41.

### CODE REVIEW

File: src/itkScancoImageIO.cxx [hunk 1]
```diff
@@ -1013,16 +1013,27 @@ ScancoImageIO::ReadImageInformation()
...
 }
 
+template <typename TBufferType>
+void 
+RescaleToHU(TBufferType * buffer, size_t size, double slope, double intercept)
+{
+  for (size_t i = 0; i < size; i++)
+  {
+    double bufferValue = static_cast<double>(buffer[i]);
+    bufferValue = bufferValue * slope + intercept;
```
Comment (dzenanz) [inline src/itkScancoImageIO.cxx:1033]: Are you forcing this computation into double type (by cast on the line before) due to fear of losing precision? This soft of operation has great potential to be vectorized, and float run about twice as fast as doubles in that case.

**Review by dzenanz — COMMENTED** [review]
Code looks good.

File: src/itkScancoImageIO.cxx (hunk 1 above)
Comment (mkuczyns) [reply src/itkScancoImageIO.cxx:1033]: That was the idea, but I can change the doubles to floats.

**Comment by dzenanz** [comment]
My amend applies ITK's style via clang-format and an update to expected baseline image, as the values are now expressed in Hounsfield units. I will wait for CI runs to complete before merging.

**Pull request merged**